src/main/java/ui/MenuControl.java

```diff
@@ -271,6 +272,20 @@ private void onBoardDelete(String boardName) {
             }
         }).collect(Collectors.toList());
     }
+    
+    public void switchBoard() {
+        if (prefs.getLastOpenBoard().isPresent() && prefs.getAllBoards().size() > 1) {
+            List<String> boardNames = new ArrayList<>(prefs.getAllBoards().keySet());
```

Comment:
This is backed by a `HashMap`, which doesn't guarantee key ordering: the boards may be cycled in a different order [over time](http://stackoverflow.com/a/2144822). You might want to change it to a `LinkedHashMap`.